index.html

```diff
@@ -204,55 +205,2367 @@ <h3>
+        <li>Set |reportItem|["{{HIDReportItem/hasPreferredState}}"] to
+        |bitfield|[5].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/hasNull}}"] to |bitfield|[6].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/isVolatile}}"] to |bitfield|[7].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/isBufferedBytes}}"] to
+        |bitfield|[8].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/usages}}"] to |usages|.
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/reportSize}}"] to
+        |globalState|["{{HIDReportItem/reportSize}}"].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/reportCount}}"] to
+        |globalState|["{{HIDReportItem/reportCount}}"].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/unitExponent}}"] to
+        |globalState|["{{HIDReportItem/unitExponent}}"].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/unitSystem}}"] to
+        |globalState|["{{HIDReportItem/unitSystem}}"].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/unitFactorLengthExponent}}"] to
+        |globalState|["{{HIDReportItem/unitFactorLengthExponent}}"].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/unitFactorMassExponent}}"] to
+        |globalState|["{{HIDReportItem/unitFactorMassExponent}}"].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/unitFactorTimeExponent}}"] to
+        |globalState|["{{HIDReportItem/unitFactorTimeExponent}}"].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/unitFactorTemperatureExponent}}"]
+        to |globalState|["{{HIDReportItem/unitFactorTemperatureExponent}}"].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/unitFactorCurrentExponent}}"] to
+        |globalState|["{{HIDReportItem/unitFactorCurrentExponent}}"].
+        </li>
+        <li>Set
+        |reportItem|["{{HIDReportItem/unitFactorLuminousIntensityExponent}}"]
+        to
+        |globalState|["{{HIDReportItem/unitFactorLuminousIntensityExponent}}"].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/logicalMinimum}}"] to
+        |globalState|["{{HIDReportItem/logicalMinimum}}"].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/logicalMaximum}}"] to
+        |globalState|["{{HIDReportItem/logicalMaximum}}"].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/physicalMinimum}}"] to
+        |globalState|["{{HIDReportItem/physicalMinimum}}"].
+        </li>
+        <li>Set |reportItem|["{{HIDReportItem/physicalMaximum}}"] to
+        |globalState|["{{HIDReportItem/physicalMaximum}}"].
+        </li>
+        <li>
+          <p>
+            If "{{HIDReportItem/usageMinimum}}" is in |localState|, then set
+            |reportItem|["{{HIDReportItem/usageMinimum}}"] to
+            |localState|["{{HIDReportItem/usageMinimum}}"].
+          </p>
+          <p>
+            Otherwise, set |reportItem|["{{HIDReportItem/usageMinimum}}"] to 0.
```

Comment:
Set it to `undefined` as it must be if `isRange` is `false?

Reply:
Fixed. I think it's sufficient to leave it unset since it should default to undefined.